helm/coredns-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 1.7.1
+appVersion: 1.8.1
```

Comment:
Wrong version or am I missing some essential magic?

Reply:
Yeah that's the wrong version. Update inbound